ED/protocol.html

```diff
@@ -686,6 +686,12 @@ <h3 property="schema:name">URI Slash Semantics</h3>
                   <p id="uri-slashes-hierarchical-identifier">The slash (<code>/</code>) character in the URI path indicates hierarchical relationship segments, and enables relative referencing [<cite><a class="bibref" href="#bib-rfc3986">RFC3986</a></cite>]. The semantics of the slash character is shared by servers and clients. Paths ending with a slash denote a container resource. [<a href="https://github.com/solid/specification/issues/35#issuecomment-547949014" rel="cito:citesAsSourceDocument">Source</a>]</p>
 
                   <p><span about="" id="server-uri-trailing-slash-distinct" rel="spec:requirement" resource="#server-uri-trailing-slash-distinct"><span property="spec:statement">If two URIs differ only in the trailing slash, and the <span rel="spec:requirementSubject" resource="#Server">server</span> has associated a resource with one of them, then the other URI <span rel="spec:requirementLevel" resource="spec:MUSTNOT">MUST NOT</span> correspond to another resource.</span></span> <span about="" id="server-uri-redirect-differing" rel="spec:requirement" resource="#server-uri-redirect-differing"><span property="spec:statement">Instead, the <span rel="spec:requirementSubject" resource="#Server">server</span> <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> respond to requests for the latter URI with a 301 redirect to the former.</span></span> [<a href="https://github.com/solid/specification/issues/107#issuecomment-567482817" rel="cito:citesAsSourceDocument">Source</a>]. <span about="" id="server-authorization-redirect" rel="spec:requirement" resource="#server-authorization-redirect"><span property="spec:statement"><span rel="spec:requirementSubject" resource="#Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> authorize prior to this optional redirect.</span></span> [<a href="https://github.com/solid/specification/issues/107#issuecomment-567454889" rel="cito:citesAsSourceDocument">Source</a>].</p>
+                  <div class="note" id="slash-and-containment" inlist="" rel="schema:hasPart" resource="slash-and-containment">
+                    <h4 property="schema:name"><span>Note</span>: Slash and containement</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>URI Slash Semantic on its own is not sufficient to determine containment. For example <code>/albums/garfield</code> does not entail <code>&lt;/albums/&gt; ldp:contains &lt;/albums/garfield&gt;</code></p>
```

Comment:
The current wording (_"URI Slash Semantic on its own is not sufficient to determine containment."_) is possibly confusing. After all, <ins>within the context of Solid</ins>, slash semantics ARE sufficient to determine containment (as of the current spec, which reads in the section on [Resource Containment](https://solidproject.org/TR/protocol#resource-containment): _"There is a 1-1 correspondence between containment triples and relative reference within the path name hierarchy."_). 

I would therefore insert the following, and possibly also move/copy the note to the Resource Containment section (in fact, it would probably be clearer if the Slash Semantics section was part of Resource Containment altogether, as that is where its impact lies.).

```suggestion
                      <p>Outside of the context of Solid, URI Slash Semantic on its own is not sufficient to determine containment. For example, without knowing whether the resource server adheres to Solid, <code>/albums/garfield</code> does not entail <code>&lt;/albums/&gt; ldp:contains &lt;/albums/garfield&gt;</code></p>
```

Reply:
Yes, I'm :-1: on the original wording in this PR, but I think with the suggestion, I'm :+1: , as it is indeed a clarification that could be useful to some.

Reply:
I find Wouter's suggestion to be generally a bit more useful here as a note. (I'm pressed on time to look at the exact location where it'd be best to put this in... but can review again after the change goes through, if any).

I'd suggest paraphrasing "outside of the context of Solid" into something more concrete, e.g., whether that's explicitly about the Solid Protocol (including its dependencies) or in particular the Storage space, or something else.

I'd also putting a bit more emphasis / clarification / distinction on the expectations/interpretations (if any) from the identifier vs. what might a representation of it describe it to be.

Pavlik, since you originally proposed, if you're on board with these, and other suggestions, could you like to or have Wouter give it another pass at the note?